lib/wire_client/base/common_message_entity.rb

```diff
@@ -0,0 +1,43 @@
+module WireClient
+  class CommonMessageEntity
```

Comment:
@earaujoassis Not sure if I follow the name. I do not see any messages returned from this class.

Also, `Account/Transaction` inheriting from `CommonMessageEntity` looks weird to me.

Account or Transaction is not a type of `CommonMessageEntity`. I would rather use `CommonMessageEntity` as a module and include in both the classes.

Wdyt?

Reply:
Yeah, the name was not the best option here. I'll update it accordingly and create a module instead of a class for it. Thanks, @abhishekjain16